cpp/src/io/parquet/writer_impl.cu

```diff
@@ -929,10 +930,10 @@ void writer::impl::write_chunk(table_view const &table, pq_chunked_state &state)
 std::unique_ptr<std::vector<uint8_t>> writer::impl::write_chunked_end(
   pq_chunked_state &state, bool return_filemetadata, const std::string &column_chunks_file_path)
 {
-  CompactProtocolWriter cpw(&buffer_);
+  CompactProtocolWriter cpw(buffer_);
```

Comment:
libcudf likes to keep in/out params as pointers.

Reply:
Was this done for performance reasons? https://github.com/rapidsai/cudf/pull/6346#issuecomment-704487207. BTW, metadata writing is not a significant part of the total time in parquet writer.

Reply:
Yeah, performance was my motivation for changing it to a reference.